externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/session/SparkSessionImpl.scala

```diff
@@ -110,12 +113,28 @@ class SparkSessionImpl(
   }
 
   override def close(): Unit = {
+    info(s"sessionId=${sessionEvent.sessionId}, " +
+      s"sessionRunTime=${formatDurationVerbose(sessionRunTime.get())}, " +
+      s"sessionCpuTime=${formatDurationVerbose(sessionCpuTime.get() / 1000000)}")
     sessionEvent.endTime = System.currentTimeMillis()
+    sessionEvent.sessionRunTime = sessionRunTime.get()
+    sessionEvent.sessionCpuTime = sessionCpuTime.get()
     EventBus.post(sessionEvent)
     super.close()
     spark.sessionState.catalog.getTempViewNames().foreach(spark.catalog.uncacheTable)
     sessionManager.operationManager.asInstanceOf[SparkSQLOperationManager].closeILoop(handle)
     sessionManager.operationManager.asInstanceOf[SparkSQLOperationManager].closePythonProcess(
       handle)
   }
+
+  val sessionRunTime = new AtomicLong(0)
+  val sessionCpuTime = new AtomicLong(0)
```

Comment:
We can move the variable definition to Line 49.

```suggestion
  private val sessionRunTime = new AtomicLong(0)
  private val sessionCpuTime = new AtomicLong(0)
```